op-conductor/conductor/service.go

```diff
@@ -584,15 +586,19 @@ func (oc *OpConductor) startSequencer() error {
 
 		if uint64(unsafeInCons.ExecutionPayload.BlockNumber)-unsafeInNode.NumberU64() == 1 {
 			// tries to post the unsafe head to op-node when head is only 1 block behind (most likely due to gossip delay)
-			if err = oc.ctrl.PostUnsafePayload(context.Background(), unsafeInCons); err != nil {
+			if err = oc.ctrl.PostUnsafePayload(ctx, unsafeInCons); err != nil {
 				oc.log.Error("failed to post unsafe head payload envelope to op-node", "err", err)
 			}
 		}
 		return ErrUnsafeHeadMismarch // return error to allow retry
 	}
 
-	if err := oc.ctrl.StartSequencer(context.Background(), unsafeInCons.ExecutionPayload.BlockHash); err != nil {
-		return errors.Wrap(err, "failed to start sequencer")
+	if err = oc.ctrl.StartSequencer(ctx, unsafeInCons.ExecutionPayload.BlockHash); err != nil {
+		if !strings.Contains(err.Error(), driver.ErrSequencerAlreadyStarted.Error()) {
+			return errors.Wrap(err, "failed to start sequencer")
```

Comment:
Typically I'd write this as:
```suggestion
			return fmt.Errorf("failed to start sequencer: %w", err)
```

Reply:
Any specific reason or pro for using `fmt.Errorf` instead of `errors.Wrap`?

My personal feeling is that although wrap is not best formatted, it does provide code trace info for ease of debugging.

Reply:
1. I'm a Java programmer and didn't know about `errors.Wrap` until I read this PR. :) 
2. We use `fmt.Errorf` everywhere else in the codebase.

I'm not strongly opinionated though.

Reply:
Updated to keep it consistent with rest of codebase.